generators/client/templates/react/package.json.ejs

```diff
@@ -27,9 +27,9 @@ limitations under the License.
     "node_modules"
   ],
   "dependencies": {
-    "@fortawesome/fontawesome-svg-core": "1.2.2",
-    "@fortawesome/free-solid-svg-icons": "5.2.0",
-    "@fortawesome/react-fontawesome": "0.1.2",
+    "@fortawesome/angular-fontawesome": "0.2.0",
```

Comment:
you mean react, instead of angular, right ?

Reply:
yeah... was too late